src/main/java/com/dataguise/saas/dto/ConnectionDTO.java

```diff
@@ -0,0 +1,49 @@
+package com.dataguise.saas.dto;
+
+import com.google.gson.Gson;
+import io.swagger.client.model.DgConnection;
+
+public class ConnectionDTO {
+
+    DgConnection connection;
+    Boolean autoSchedule;
+    String agentName;
+    String agentHostName;
+
+    public DgConnection getConnection() {
+        return connection;
+    }
+
+    public void setConnection(DgConnection connection) {
+        this.connection = connection;
+    }
+
+    public Boolean getAutoSchedule() {
+        return autoSchedule;
+    }
+
+    public void setAutoSchedule(Boolean autoSchedule) {
+        this.autoSchedule = autoSchedule;
+    }
+
+    public String getAgentName() {
+        return agentName;
+    }
+
+    public void setAgentName(String agentName) {
+        this.agentName = agentName;
+    }
+
+    public String getAgentHostName() {
+        return agentHostName;
+    }
+
+    public void setAgentHostName(String agentHostName) {
+        this.agentHostName = agentHostName;
+    }
+
+    @Override
```

Comment:
**Reporter**: Checkstyle    
    **Rule**: com.puppycrawl.tools.checkstyle.checks.design.DesignForExtensionCheck
    **Severity**: ERROR
    **File**: /work/src/main/java/com/dataguise/saas/dto/ConnectionDTO.java L45
    
    Class &#39;ConnectionDTO&#39; looks like designed for extension (can be subclassed), but the method &#39;toString&#39; does not have javadoc that explains how to do that safely. If class is not designed for extension consider making the class &#39;ConnectionDTO&#39; final or making the method &#39;toString&#39; static/final/abstract/empty, or adding allowed annotation for the method.
    
*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a853270267>*